Dockerfile

```diff
@@ -1,26 +1,41 @@
-FROM debian:bookworm
+FROM nvidia/cuda:12.4.1-devel-ubuntu22.04
 
-# install Octopus (latest stable or develop) on Debian
+# ---------------------------------------------------------------------
+# Install Octopus (latest stable or develop) on CUDA container
+# ---------------------------------------------------------------------
 
 # the version to install (latest stable or develop) is set by buildarg VERSION_OCTOPUS
 # the development version of octopus is hosted on the branch "main" in the official repository.
-ARG VERSION_OCTOPUS=main
+ARG VERSION_OCTOPUS=14.0
```

Comment:
Is there a particular reason we are setting the default value of the octopus version? The problem is that of maintainability. one would have to update this every time we have new octopus release. `main` is a generic branch and thus seems like a better option for defaults

Reply:
This is just for our book keeping purposes. We archive recipes for each software and each version we have on our clusters. This does not have to be pushed to your repo. :)